data/src/main/java/com/teamagam/gimelgimel/data/response/entity/GGResponse.java

```diff
@@ -1,12 +1,12 @@
-package com.teamagam.gimelgimel.data.message.entity;
+package com.teamagam.gimelgimel.data.response.entity;
 
 import com.google.gson.annotations.SerializedName;
-import com.teamagam.gimelgimel.data.message.entity.visitor.IMessageDataVisitable;
+import com.teamagam.gimelgimel.data.response.entity.visitor.IResponseVisitable;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.util.Date;
 
-public abstract class MessageData<T> implements IMessageDataVisitable {
+public abstract class GGResponse<T> implements IResponseVisitable {
```

Comment:
The response is from the server and not from GG (the application), so I think the GG prefix is confusing.
ServerResonse / RemoteResponse / Response / ??? would be better